### PR TITLE
Fermdem/ Improvements and refactoring of the App files

### PR DESCRIPTION
**Pull request by AlejandroCornejo**

### CODE REVIEW

File: applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp [hunk 1]
```diff
@@ -165,14 +160,12 @@ class StressToNodesProcess : public Process
         rmax_id = aux;
     }
 
-    //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
   protected:
     // Member Variables
-    ModelPart &mr_model_part;
+    ModelPart& mr_model_part;
```
Comment (loumalouomega) [inline applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp]: Should be mrModelPart

**Review by loumalouomega — CHANGES_REQUESTED** [review]

File: applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp (hunk 1 above)
Comment (AlejandroCornejo) [reply applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp]: done thx

Comment (loumalouomega) [reply applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp]: Done¿?

Comment (AlejandroCornejo) [reply applications/FemToDemApplication/custom_processes/stress_to_nodes_process.hpp]: I was waiting for more things before pushing hehe

**Review by loumalouomega — DISMISSED** [review]

**Review by loumalouomega — APPROVED** [review]
Ok

**Pull request merged**